lib/index.js

```diff
@@ -152,6 +153,16 @@ export const Dataviz = function (options = {}) {
     this.config.type = convertChartTypes(this.config.type);
   }
 
+  // overwriting options for range chart
+  if(this.config.type.includes('-range')) {
+    this.config.range = true;
+    this.config.type = this.config.type.replace('-range', '');
+    this.config.legend.show = false;
+    this.config.labels = ['Max', 'Min'];
+    this.config.stacking = 'normal';
+    this.config.colors = ['#ff0000', '#ff0000'];
```

Comment:
those colors should be inherited from the main config (or custom colors passed by user)